### PR TITLE
Fix callback/selective task print being long

### PR DESCRIPTION
**Pull request by AidanHouck**
##### SUMMARY
When printing the task name section of command output, previously the line was always 3 chars longer than `line_length` which is meant to limit width. This was due to the length check simply looking at the `task_name`, and not including the 3 formatting characters added in the string directly (`# {0} `). 

To fix this, I pulled these characters out into a separate variable then did the length check on that.

bugfixes:
  - selective - fix task name output using improper length check and going over the defined width limit

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
selective

##### ADDITIONAL INFORMATION
Note that there are many other checks in this plugin that also do not properly follow the `line_length` variable for one reason or another. I will submit fixes for those later in a separate PR as they are not as trivial to resolve.

### CODE REVIEW

File: plugins/callback/selective.py [hunk 1]
```diff
@@ -115,8 +115,8 @@ def _print_task(self, task_name=None):
             line_length = 120
             if self.last_skipped:
                 print()
-            msg = colorize("# {0} {1}".format(task_name,
-                                              '*' * (line_length - len(task_name))), 'bold')
+            line = "# {0} ".format(task_name)
+            msg = colorize("{0}{1}".format(line, '*' * (line_length - len(line))), 'bold')
```
Comment (russoz) [inline plugins/callback/selective.py:119]: Wouldn't it be simpler to go

```suggestion
            msg = colorize("{0}{1}".format(line, '*' * (line_length - len(line) - 3)), 'bold')
```
?

**Comment by russoz** [comment]
Hi @AidanHouck ,

Thank you for the contribution! Please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

File: plugins/callback/selective.py (hunk 1 above)
Comment (AidanHouck) [reply plugins/callback/selective.py:119]: Yes, however if somebody ever comes along and changes the formatting and it uses a different number of characters they would need to manually update the magic number in that comparison. I figured this was better and also felt more inline with the style of the rest of the file (e.g. all of the `_print_host_or_item()` func)

Comment (russoz) [reply plugins/callback/selective.py:119]: Fair enough

File: changelogs/fragments/7374-fix-selective-callback-taskname-length.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "selective callback plugin - fix length of task name lines in output always being 3 characters longer than desired."
```
Comment (russoz) [inline changelogs/fragments/7374-fix-selective-callback-taskname-length.yml:2]: Please add a reference to the ticket as in:
```suggestion
  - "selective callback plugin - fix length of task name lines in output always being 3 characters longer than desired (https://github.com/ansible-collections/community.general/pull/7374)."
```

Comment (AidanHouck) [reply changelogs/fragments/7374-fix-selective-callback-taskname-length.yml]: Done

**Comment by felixfontein** [comment]
CI failures are unrelated and should get fixed by #7385.

**Review by russoz — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@AidanHouck thanks for your contribution!
@russoz thanks for reviewing!

**Pull request merged**